tools/auto_tuner/include/gemm_tuner.hpp

```diff
@@ -0,0 +1,297 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) 2018 Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename gemm_tuner.hpp
+ *
+ **************************************************************************/
+
+#include <iostream>
+#include <random>
+#include <vector>
+
+#include <chrono>
+#include <cmath>
+#include <functional>
+#include <numeric>
+
+#include "reference_gemm.hpp"
+#include "sycl_blas.hpp"
+
+using namespace cl::sycl;
+using namespace blas;
+
+#ifdef linux
+#include <stdio.h>
+#include <unistd.h>
+
+// A status bar thing
```

Comment:
This is a silly little thing, let me know if it should stay.

Reply:
please remove that :)

Reply:
Is gone now. ;_;